cscs-checks/prgenv/gpu_direct_acc.py

```diff
@@ -29,14 +29,17 @@ def __init__(self, **kwargs):
             self.num_tasks_per_node = 8
 
         self.sourcepath = 'gpu_direct_acc.F90'
-        self.sanity_patterns = sn.assert_found(r'Result :\s+OK', self.stdout)
+        self.sanity_patterns = sn.all([sn.assert_found(r'GPU with OpenACC',
+                                                       self.stdout),
+                                       sn.assert_found(r'Result :\s+OK',
+                                                       self.stdout)])
```

Comment:
Format this as follows; it's more readable:

```python
        self.sanity_patterns = sn.all([
            sn.assert_found(r'GPU with OpenACC', self.stdout),
            sn.assert_found(r'Result :\s+OK',self.stdout)
        ])
```

Reply:
done